configure.py

```diff
@@ -384,12 +384,16 @@ def set_action_env_var(environ_cp,
 def convert_version_to_int(version):
   """Convert a version number to a integer that can be used to compare.
 
+  Version strings of the form X.YZ and X.Y.Z-xxxxx are supported. The
```

Comment:
The current versions of bazel do have a leading zero - mine is '0.5.2-homebrew'.

I know what you mean though - In the past I have solved this problem by converting X.Y.Z into X.YZ, where Y and Z are formatted %04u.  

I'm not really trying to solve that problem though.  I could raise an issue for that if you would like.